entity-types/infra-awsvpn/summary_metrics.yml

```diff
@@ -0,0 +1,18 @@
+providerAccountName:
+  title: AWS account
+  unit: STRING
+  tag:
+    key: providerAccountName
+region:
+  title: Region
+  unit: STRING
+  tag:
+    key: aws.awsRegion
+tunnelDataIn:
+  goldenMetric: tunnelDataIn
+  title: Tunnel data in
+  unit: COUNT
+tunnelDataOut:
+  goldenMetric: tunnelDataOut
+  title: Tunnel data Out
+  unit: COUNT
```

Comment:
NIT: Would it be better to use some more precise unit like https://github.com/newrelic/entity-definitions/blob/main/docs/entities/golden_metrics.md#metric-unit (also for the dashboard)